app/views/unattended/provisioning_templates/finish/windows_default_finish.erb

```diff
@@ -44,89 +45,104 @@ description: |
 <% if @host.pxe_build? %>
   set ctr=0
   set nettimeout=10
+<% end -%>
 
-  (echo Updating time)
-  (sc config w32time start= auto)
-  sc start w32time
-  ::ipconfig /renew
+  <%= snippet 'windows_network' %>
 
   <% if host_param('ntpServer') %>
-    echo setting time server
+    echo Setting time server
     w32tm /config /manualpeerlist:<%= host_param('ntpServer') %> /syncfromflags:manual /update
   <% end %>
 
-  echo sync time
+  echo Syncing time
   w32tm /resync
   w32tm /resync
 
   <% if host_param('computerDomain') -%>
     <% if host_param('domainAdminAccount').present? && host_param('domainAdminAccountPasswd').present? -%>
-      echo performing secure domain join
+      echo Performing secure domain join
       powershell.exe -OutputFormat text -command Add-Computer -DomainName '<%= host_param('computerDomain') -%>' -Credential (New-Object -TypeName System.Management.Automation.PSCredential '<%= host_param('domainAdminAccount') -%>', (ConvertTo-SecureString -String '<%= host_param('domainAdminAccountPasswd') -%>' -AsPlainText -Force)) <% if host_param('computerOU').present? -%>-OUPath '<%= host_param('computerOU') -%>'<% end -%>
     <% else %>
       <% if host_param('machinePassword').present? %>
-        echo performing unsecure domain join
+        echo Performing unsecure domain join
         powershell.exe -OutputFormat text -command Add-Computer -Domain '<%= host_param('computerDomain') -%>' -Options UnsecuredJoin,PasswordPass -Credential (New-Object -TypeName System.Management.Automation.PSCredential $null, (ConvertTo-SecureString -String '<%= host_param('machinePassword') -%>' -AsPlainText -Force))
       <% end %>
     <% end %>
   <% end %>
 
+  <%#
+  Adjust power setting to 'High performance'
+  -%>
+  powershell /c "powercfg /setactive 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c"
+
   <% if host_param('localAdminAccountDisabled') %>
     echo Disabling %tempAdminUser%
     net user %tempAdminUser% %tempAdminUser% /active:no
   <% end %>
 
-  <% if host_param('ansible_port') == 5985 or host_param('ansible_winrm_scheme') == 'http' %>
-    cmd /c winrm set winrm/config/service @{AllowUnencrypted="true"}
+  <% if host_param('http-proxy').present? -%>
+    cmd /C "netsh winhttp set proxy <%= host_param('http-proxy') %>:<%= host_param('http-proxy-port') %>"
+  <% end %>
+
+  <% unless host_param('computerDomain') %>
+    powershell /c "Get-NetConnectionProfile -InterfaceAlias \"Ethernet0\" | Set-NetConnectionProfile -NetworkCategory <%= network_location %>"
   <% end %>
 
-  <% if host_param('ansible_winrm_transport') == 'basic' %>
+  <% if host_param('ansible_user').present? %>
+    <% if host_param('create_ansible_user') %>
```

Comment:
If you use `host_param_true?` you can set it to `true` on the host group level and `false` on the host level.
```suggestion
    <% if host_param_true?('create_ansible_user') %>
```